datafusion/sql/tests/cases/plan_to_sql.rs

```diff
@@ -1746,3 +1749,153 @@ fn test_unparse_subquery_alias_with_table_pushdown() -> Result<()> {
     assert_eq!(sql.to_string(), expected);
     Ok(())
 }
+
+#[test]
+fn test_unparse_left_anti_join() -> Result<()> {
+    let schema = Schema::new(vec![
+        Field::new("c", DataType::Int32, false),
+        Field::new("d", DataType::Int32, false),
+    ]);
+
+    // LeftAnti Join: t1.c = __correlated_sq_1.c
```

Comment:
Could you add the original SQL in the comment? It would be nice for other people to know where the plan comes from.